examples/ssx-test-dapp/src/App.js

```diff
@@ -114,6 +111,10 @@ function App() {
       modules.encryption = true;
     }
 
+    if (encryptionEnabled === "On") {
```

Comment:
why `"On"` instead of `true`?

Reply:
We did this because this is a select with multiple options. We just use it on the test-dapp, because there are multiple options to configure SSX